arch/xtensa/src/esp32/esp32_rt_timer.c

```diff
@@ -302,7 +303,7 @@ static int rt_timer_thread(int argc, FAR char *argv[])
 
   while (1)
     {
-      /* Waiting for timers timeout */
+      /* Waiting for the timers timeouts */
```

Comment:
```suggestion
      /* Waiting for all timers to time out */
```
Suggestion to apply the same modification from other file.